sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/ServiceBusAttribute.cs

```diff
@@ -55,5 +65,13 @@ public ServiceBusAttribute(string queueOrTopicName, ServiceBusEntityType entityT
         /// Value indicating the type of the entity to bind to.
         /// </summary>
         public ServiceBusEntityType EntityType { get; set; }
+
+        /// <summary>
+        /// This property indicates whether the string being sent from out of process functions
+        /// includes the message metadata. By default this is false, meaning the string represents
+        /// only the message body.
+        /// </summary>
+        [JsonProperty]
+        internal bool IncludeMetadata { get; set; }
```

Comment:
I _think_ this should work even though it is internal. My understanding is that the attributes for out of process .NET and for other languages get serialized using NewtonSoft, sent to the host process, and then deserialized into this model also using NewtonSoft.

/cc @fabiocav @mattchenderson

Reply:
I dug into this, and the property does need to be public as it is consumed here - https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeConverter.cs#L72